lib/util.js

```diff
@@ -785,6 +792,10 @@ exports.isPrimitive = isPrimitive;
 
 exports.isBuffer = Buffer.isBuffer;
 
+function isProxy(p) {
+  return binding.isProxy(p);
+}
+exports.isProxy = isProxy;
```

Comment:
+1 ... Forgot to add the doc for it. Will add soon!
On Apr 29, 2016 6:29 AM, "Colin Ihrig" <notifications@github.com> wrote:

> In lib/util.js
> <https://github.com/nodejs/node/pull/6465#discussion_r61577439>:
>
> > @@ -785,6 +792,10 @@ exports.isPrimitive = isPrimitive;
> >
> >  exports.isBuffer = Buffer.isBuffer;
> >
> > +function isProxy(p) {
> > +  return binding.isProxy(p);
> > +}
> > +exports.isProxy = isProxy;
>
> If you export this, it becomes public API, and should be documented.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6465/files/3d4ff4311a3cf1ff756413756ffc859a663aba99#r61577439>
>